ee/clickhouse/sql/session_recording_events.py

```diff
@@ -22,7 +23,7 @@
 SESSION_RECORDING_EVENTS_TABLE_SQL = (
     SESSION_RECORDING_EVENTS_TABLE_BASE_SQL
     + """PARTITION BY toYYYYMMDD(timestamp)
-ORDER BY (team_id, toHour(timestamp), session_id, timestamp, uuid)
+ORDER BY (team_id, toHour(timestamp), session_id, window_id, timestamp, uuid)
```

Comment:
So a change here cannot be applied without reingesting the data.

Given that we expect every session_id to have a small amount of events, I don't think it's worth it to change it and either:
1. Go out of sync with the cloud schema
2. Spend a ton of time manually re-schemaing the table on cloud.

Reply:
Ooo interesting - makes sense to me!